src/app/App.tsx

```diff
@@ -12,14 +13,16 @@ import { Transactions } from './transactions'
 
 const mapActionToProps = {
   initializeElectra: ElectraActions.initializeElectra,
-  generateHDWallet: ElectraActions.generateHDWallet
+  generateHDWallet: ElectraActions.generateHDWallet,
+  getGlobalBalance: OverviewActions.getGlobalBalance
 }
 
 @connect(null, mapActionToProps)
 export default class App extends React.Component<any, any> {
   componentWillMount() {
     this.props.initializeElectra() // initialize electraJs object
     this.props.generateHDWallet() // generate HD wallet
+    this.props.getGlobalBalance() // get global balance
```

Comment:
You can call this in it's own feature/page e.g. in overview feature's `container.tsx`